ts-relayer-tests/src/ics721.spec.ts

```diff
@@ -186,17 +276,135 @@ test.serial("transfer NFT", async (t) => {
   t.is(osmoAddr, tokenOwner.owner);
 });
 
+test.serial(
+  "transfer NFT with osmo outgoing and wasm incoming proxy",
+  async (t) => {
```

Comment:
NEW: added ts-relayer integration test for:
- ics721 handles incoming msg from outgoing proxy properly
- ics721 passes ibc packet to incoming proxy
  - success: in case packet is coming from WLed channel
  - fail: in case channel is not WLed -> NFT returned back to onwer on source chain